geometric/params.py

```diff
@@ -289,11 +310,34 @@ def parse_optimizer_args(*args):
     grp_jobtype.add_argument('--transition', type=str2bool, help='Provide "yes" to Search for a first order saddle point / transition state.\n ')
     grp_jobtype.add_argument('--meci', type=str, nargs="+", help='Provide second input file and search for minimum-energy conical\n '
                              'intersection or crossing point between two SCF solutions (TeraChem and Q-Chem supported).\n'
-                             'Or, provide "engine" if the engine directly provides the MECI objective function and gradient.')
+                             'Or, provide "engine" if the engine directly provides the MECI objective function and gradient.\n')
     grp_jobtype.add_argument('--meci_sigma', type=float, help='Sigma parameter for MECI penalty function (default 3.5).\n'
                             'Not used if the engine computes the MECI objective function directly.\n ')
     grp_jobtype.add_argument('--meci_alpha', type=float, help='Alpha parameter for MECI penalty function (default 0.025).\n'
-                             'Not used if the engine computes the MECI objective function directly.')
+                             'Not used if the engine computes the MECI objective function directly.\n ')
```

Comment:
Similarly, should NEB use the same command line "menu" as geometry optimization or should it be a completely different menu?  We should think about this as well.  I currently think it should be a different menu, in order to avoid confusing or overlapping options.  So the entry point would be something like `geometric-neb` and it would have these options here, and we can copy over any ones we need from `geometric-optimize`.

Reply:
Done.